CMakeLists.txt

```diff
@@ -177,8 +177,14 @@ install(
 # Process subdirectories
 add_subdirectory(doc)
 add_subdirectory(src)
-add_subdirectory(tests)
-add_subdirectory(samples)
+if (NOT (DEFINED XERCES_BUILD_FUZZERS))
+  add_subdirectory(tests)
+  add_subdirectory(samples)
+endif()
+
+if ((DEFINED XERCES_BUILD_FUZZERS) OR (DEFINED XERCES_BUILD_FOR_OSS_FUZZ))
```

Comment:
Please always include all subdirectories, and place the above conditionals around the logic inside the file.  This ensures the logic is evaluated whatever the cmake options, and helps to prevent regressions.

Reply:
Additionally, rather than use XERCES_BUILD_FUZZERS, why not define an option `fuzzers` with `option()` defaulting to off, then you can just use `-Dfuzzers=ON`?

Reply:
Will do that, thanks!